roadmap/roadmap.json

```diff
@@ -1176,6 +1150,32 @@
             "docs": "",
             "is_community": false,
             "in_beta": false
+          },
+          {
+            "title": "Public specification for GPU-enabled nodes",
+            "overview": "The ICP community needs to agree and adopt the hardware specification for replica nodes with GPUs.",
+            "description": "The ICP community needs to agree and adopt the hardware specification for replica nodes with GPUs.",
+            "status": "",
+            "milestone_id": "Gyrotron",
```

Comment:
Would it make sense to push this and the next feature to another milestone (that comes after Gyrotron)?

Reply:
Yes, this is the list of future features. Here's how it looks like:

<img width="1130" alt="image" src="https://github.com/user-attachments/assets/3f603e21-5857-4cb1-a791-6421f2e14bed">

Does this map to what you're thinking as well, or did I misunderstand?

Reply:
Looks good!